### DISCUSSION TITLE
Call to undefined method ReflectionUnionType::getName()

### DISCUSSION THREAD

**Discussion by sakanjo** [livewire/livewire#6361] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.9

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

```php
<?php

namespace App\Livewire;

use Livewire\Component;

class CreateInt extends Component
{
    public string|int $age = 10; # Here's the problem

    public function save(): void
    {
        back()->with('message', 'The integer was saved!');
    }

    public function render()
    {
        return view('livewire.create-int');
    }
}

```

```html
<form wire:submit.prevent="save">
    <input wire:model="age" type="number" />

    <div>
        <button type="submit">Save</button>
    </div>
</form>

```

### Additional information

![example](https://github.com/livewire/livewire/assets/121197517/8af4fb08-bd93-40df-af01-173b8b0529ff)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by sakanjo** [comment]
#6362 PR

**Reply by nuernbergerA** [reply]
just do
```php
public ?int $age = 10;
```
and you are good

**Reply by sakanjo** [reply]
That's just an example the main problem is on https://github.com/filamentphp/filament/issues/7778.
